config/cable.yml

```diff
@@ -1,10 +1,11 @@
 development:
-  adapter: async
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
Do we actually run a redis instance now? Or was this running for other reasons?

Reply:
Hm weird - this must be a Rails 7 default as well. Good catch!

I'll double check that the app builds/runs even if it's not installed.

Reply:
I went ahead and removed the dependency. Thanks for the callout!